doc/skill_require_db.txt

```diff
@@ -104,13 +104,13 @@ RequiredAmmoAmount: Amount of ammo needed to use the skill.
 
 RequiredState: The active 'State' needed to use the skill.
 
-none = Nothing special.
+none = Nothing.
 hidden = Requires hidden status by using Hiding, Cloaking, or Chasewalk.
 riding = Requires the player to ride either a Peco or a Dragon.
```

Comment:
This state only checks for Peco or Dragon specifically:
```c
		case ST_RIDING:
			if(!pc_isriding(sd) && !pc_isridingdragon(sd)) {
				clif_skill_fail(sd,skill_id,USESKILL_FAIL_LEVEL,0);
				return false;
			}
			break;
```
The cash shop mounts don't have any specific skill checks.